core/Inst.hpp

```diff
@@ -485,6 +506,12 @@ namespace olympia
             1; // start at 1 because the uop count includes the parent instruction
         uint64_t uop_count_ = 0;
         VCSRs VCSRs_;
+        uint32_t eew_;
```

Comment:
I would move these values into the `VCSRs` structure and rename it to something like `VectorConfig` or `VectorState`.